tools/scripts/BuildZippedCLI.ps1

```diff
@@ -95,8 +101,6 @@ function Create-Zipfile([string]$srcDir, [string]$otherSignedSrcDir, [string]$ap
     Prepare-Zipfile $srcDir $otherSignedSrcDir $scratch $patternsToRemove $otherSignedAssemblies
 
     Create-Archive $scratch $zipFile
-
-    Remove-Item $scratch -Force -Recurse
```

Comment:
We need to leave the directory with contents for the task to scan. Since script only runs when a specific environment variable is set (CreateAxeWindowsZippedCLI) and the validation task runs at the end of our pipeline, leaving the files in the system temp directory seemed acceptable to me.